CMakeLists.txt

```diff
@@ -341,6 +356,11 @@ if( BUILD_TESTING AND TRACCC_BUILD_TESTING )
    add_subdirectory( tests )
 endif()
 
+# Set up the benchmark(s).
+if( TRACCC_BUILD_BENCHMARK )
+   add_subdirectory( benchmarks )
```

Comment:
Similar to what is done for the examples and the tests, you should add a check for `TRACCC_BUILD_IO` here as well.

Reply:
OK